src/superphot_plus/lightcurve.py

```diff
@@ -329,7 +329,7 @@ def from_file(cls, filename, ref_band="r", t0_lim=None, shift_time=True):
         arr = None
         property_dict = {}
         for k in npy_array.files:
-            if k == "lcs":
+            if k == "lcs" or k == "arr_0": #hotfix to handle old LC format
```

Comment:
Can we also set the light curve name in case we're using the old format? Maybe to something like `filename.split('.')[0]` so that we get the light curve file name without the npz extension? Otherwise when we try to save its generated posteriors to a file we'll get an output file name that contains None.